src/praisonai-agents/praisonaiagents/agent/chat_mixin.py

```
@@ -135,6 +135,50 @@ def _build_system_prompt(self, tools=None):
+                            desc += f" ({target.platform}, home channel)"
+                        elif target.kind == "alias":
+                            desc += f" ({target.platform}:{target.channel_id})"
+                        target_descriptions.append(desc)
+                    
+                    if target_descriptions:
+                        context_parts.append(
+                            f"Reachable delivery targets: {', '.join(target_descriptions)}."
+                        )
+                
+                if context_parts:
+                    system_prompt += f"\n\n## Session Context\n" + "\n".join(context_parts)
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)